### DISCUSSION TITLE
Creating Postgres data source becoming MySQL

### DISCUSSION THREAD

**Discussion by scmsystm** [illacloud/illa-builder#1345] [Q&A] [answered]
Hello, Im currenlty trying ILLA on local machine, but when I try to create postgres datasource it can't test connection or connect and type was MySQL.
Running ILLA builder on docker multipass windows 10 host.
![scrnli_4_4_2023_22-07-37](https://user-images.githubusercontent.com/1313750/229836706-a33a7763-5cc9-4f85-a844-c012e76149b2.gif)

**Comment by naj1n** [comment]
We apologize for this issue. We have made relevant updates regarding this issue and hope you can test it again.

**Reply by scmsystm** [reply, marked as answer]
> We apologize for this issue. We have made relevant updates regarding this issue and hope you can test it again.

yeah it seem already fixed now, thank you.

**Reply by naj1n** [reply]
Thanks for your feedback. Feel free to contact us if you have any questions.
